docs/saf-authorized-endpoints.md

```diff
@@ -0,0 +1,101 @@
+
+# Secured endpoints with SAF resource 
+
+Any endpoint can have limited access by checking SAF resource of logged user. This feature is implemented in module
+`apiml-security-common`. Therefore, it can be used in each application with this dependency.
+
+Verification of SAF resource can be provided via three providers:
+- _the highest priority_: REST endpoint call (ZSS or similar one)
+- JZOS
+- _the lowest priority_: dummy implementation (defined in a file)
+
+**note**: The first available by priority will be used.
+
+## Checking providers
+
+### REST endpoint call
+
+This provider is once way how to enable the feature outside the mainframe (ie. running in Docker).
+
+- Method: `GET`
+- URL: `{base path}/{userId}/{class}/{entity}/{level}`
+- Response:
+```json5
+    {
+        "authorized": "{true|false}",
+        "error": "{true|false}",
+        "message": "{message}"
+    }
+```
+
+**note**: see also ZSS implementation https://github.com/zowe/zss/blob/master/c/authService.c
+
+#### Configuration
+
+`apiml.security.safEndpoint.enabled`
```

Comment:
Are you talking about the ZSS endpoint or some specific authorization endpoint in the API ML?